packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -63,17 +64,41 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                 is_left_swipe
                 right_hidden_component={
                     <React.Fragment>
-                        <div className='p2p-my-ads__table-popovers__edit' onClick={onClickEdit}>
-                            <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
-                        </div>
-                        <div className='p2p-my-ads__table-popovers__activate'>
-                            <Icon
-                                icon='IcArchive'
-                                custom_color='var(--general-main-1)'
-                                size={14}
-                                onClick={onClickActivateDeactivate}
-                            />
-                        </div>
+                        {is_advert_active ? (
+                            <div
+                                className='p2p-my-ads__table-popovers__edit'
+                                onClick={
+                                    enable_action_point && floating_rate_store.rate_type !== rate_type
+                                        ? onClickSwitchAd
+                                        : onClickEdit
+                                }
+                            >
+                                <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
```

Comment:
I believe the default size in `Icon` is already set to 16.

Reply:
Will remove the prop